### ISSUE TITLE
Node.js 12 support

### ISSUE THREAD

**Issue by ai** [hegemonic/requizzle#5]
After updating to Node.js 12 I start to get the error:

```
$ jsdoc -d docs/ *.js react/*.js
/home/travis/build/ai/storeon/node_modules/requizzle/lib/requizzle.js:45
	if (lookupPaths[0] === targetPath && lookupPaths[1].length === 0) {
	               ^
TypeError: Cannot read property '0' of null
    at isNativeModule (/home/travis/build/ai/storeon/node_modules/requizzle/lib/requizzle.js:45:17)
    at Requizzle.requizzle (/home/travis/build/ai/storeon/node_modules/requizzle/lib/requizzle.js:82:6)
    at infectProxy (/home/travis/build/ai/storeon/node_modules/requizzle/lib/loader.js:82:28)
    at Module.targetModule.require (/home/travis/build/ai/storeon/node_modules/requizzle/lib/loader.js:101:11)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/home/travis/build/ai/storeon/node_modules/jsdoc/lib/jsdoc/fs.js:7:10)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Object.load (/home/travis/build/ai/storeon/node_modules/requizzle/lib/loader.js:112:15)
```

https://travis-ci.org/ai/storeon/builds/523619135

Hope it is not a JSDoc error since JSDoc has no maintenance anymore.

**Comment by jgravois** [comment]
it _looks like_ https://github.com/hegemonic/requizzle/pull/4 already attempts to address this.

**Comment by ai** [comment]
@hegemonic seems like we found a way to fix this issue

**Comment by Maledong** [comment]
@ai：Is this fixed or anything to fix? Would you mind sharing your suggestion? :)

**Comment by ai** [comment]
@Maledong #4 is a PR, which probably will fix this issue. It is not merged yet.

**Comment by Maledong** [comment]
Thanks!

**Comment by 75lb** [comment]
[jsdoc2md](https://github.com/jsdoc2md/jsdoc-to-markdown) is currently broken in node v12 until this issue is resolved.

**Comment by hegemonic** [comment]
Fixed by #4. Apologies for the delay.

I'll release a new version of this module soon, followed by a new version of JSDoc. I'm swamped right now, so it'll probably take a few days. In the meantime, thanks for your patience.
